daisy_workflows/image_import/debian/translate_debian.wf.json

```diff
@@ -77,7 +77,8 @@
           }
         }
       ],
-      "Timeout": "60m"
+      "Timeout": "60m",
+      "ErrorDescription": "This might indicate the disk is not bootable in Google Compute Engine. Make sure the disk satisfies the requirements documented at https://cloud.google.com/compute/docs/import/importing-virtual-disks."
```

Comment:
is it possible to catch the bootable issue other than waiting for timeout? I think response of instance.start API should be able to give us some info about the boot.

Reply:
We don't call instance.start, instances are auto started on create. 

Also, instance.create and instance.start only return errors, not detailed status on whether OS has booted. GCE doesn't know if OS has booted or not or if there's an OS at all running on the instance.